Sources/ApodiniUtils/Shell.swift

```diff
@@ -27,17 +27,13 @@ public enum ShellCommand {
 /// A helper function to run custom shell commands, e.g. on app launch
 @discardableResult
 public func runShellCommand(_ command: ShellCommand) -> String {
-    shell(command.method)
-}
-
-@discardableResult
-private func shell(_ command: String) -> String {
+    //(try? ChildProcess.runZshShellCommandSync(command.method).output) ?? ""
     let task = Process()
     let pipe = Pipe()
-
+    
```

Comment:
Isn't there a difference between trailing whitespace at the end of an otherwise non-empty line, and lines that consist only of whitespace? In the first case, you obviously don't want that,  but in the second case I feel like the indentation of an empty line should match the indentation of the lines around it, right?